dataclients/kubernetes/definitions/routegroups.go

```diff
@@ -426,6 +407,20 @@ func (r *routeSpec) validate(hasDefault bool, backends map[string]bool) error {
 	return nil
 }
 
-func (r *routeSpec) uniqueMethods() []string {
-	return uniqueStrings(r.Methods)
+func (r *RouteGroupItem) validate() error {
```

Comment:
can you add a TODO comment that we need to pass namespace/name in all errors?

Reply:
Done